### ISSUE TITLE
Check root types on production minerva for all of the life stages (WBls, ZFS, XAO, Theiler)

### ISSUE THREAD

**Issue by vanaukenk** [geneontology/minerva#466]
On production Noctua, life stage terms don't appear as expected on the Noctua form table.  The display works correctly on noctua-dev, though.  (See: https://github.com/geneontology/minerva/issues/428)

@balhoff  Can you check to see if the life stages are correctly typed as UBERON:0000105 on production minerva?

Thank you!

**Comment by balhoff** [comment]
@vanaukenk is this still a problem after the last restart? They look okay in the ontology.

**Comment by vanaukenk** [comment]
@balhoff Here's what I see in the Noctua form vs the graph editor for life stage terms:

![image](https://user-images.githubusercontent.com/1730534/159789586-ff543341-c6c8-4dfa-ac8e-21f709d2d39e.png)

![image](https://user-images.githubusercontent.com/1730534/159789834-0518a072-c7fd-4cc7-b322-5e3da4a43f27.png)

The graph editor displays the 'existence starts and ends during' extension but it's not in the form which @tmushayahama explained to me is because of missing root types in production.

**Comment by tmushayahama** [comment]
@balhoff yes, the root type (is_a_closure) entity is empty. I haven't checked this morning.

**Comment by balhoff** [comment]
@kltm could you check the git revision on the master deployment? I'm adding this to the status endpoint so that this will be easy in the future (along with the ontology version IRI).

**Comment by kltm** [comment]
On noctua production, I have:
```
git log
commit 6c9dfd230ba5d2acae4140ef1bd94a1e9330868e
Merge: 735ab0f cafd91f
Author: kltm <github.com@genkisugi.net>
Date:   Wed Dec 15 11:10:39 2021 -0800

    Merge pull request #439 from geneontology/log4j
    
    Use log4j 2.16.0.

```

**Comment by balhoff** [comment]
Thanks! This indicates that what we need for this issue is to update production Minerva to the latest code from `master`.

**Comment by vanaukenk** [comment]
This appears to be working fine, except for the Theiler stages, but there is a separate ticket for that:  https://github.com/geneontology/minerva/issues/449

Internal test model:  http://noctua.geneontology.org/workbench/noctua-form/?model_id=gomodel%3A6258b8f300000405
